project/src/main/java/com/google/sps/servlets/QueryServlet.java

```diff
@@ -20,10 +20,15 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+
+import com.google.sps.data.Place;
 import com.google.sps.data.Places;
 import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableList;
 import com.google.gson.Gson;
+import com.google.maps.model.LatLng;
 import com.google.sps.data.PlacesFetcher;
+import com.google.sps.data.UserPreferences;
 
 /**
  * A servlet that handles the user query. Currently accepts no input, and responds with a list of
```

Comment:
I think this comment needs to be updated.

Reply:
done, thanks